pydra/engine/core.py

```diff
@@ -504,15 +504,24 @@ def get_input_el(self, ind):
             return None, inputs_dict
 
     def to_job(self, ind):
-        """Run interface one element generated from node_state."""
+        """ Pickling tasks and inputs, so don't have to load the input to teh memory """
```

Comment:
```suggestion
        """Creates a standalone job by pickling to the task and inputs to disk. 
        
        This reduces memory consumption locally as a deepcopy is avoided. In future 
        work this will be optimized between storage vs memory consumption.
        """
```

Reply:
Seems like this function can be simply reduced to pickling the task and input without any consideration `ind` and the function that calls this can deal with indices. `ind` appears mostly as a passthrough.